spec/models/maths_and_physics/dqt_record_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe MathsAndPhysics::DQTRecord do
+  let(:maths_jac_code) { "G100" }
```

Comment:
I think we should increase the coverage for the different variations of valid JACs codes, right now we are only testing two variations, which feels a bit thin. How about we use a list of valid and list of invalid ones and then iterated through those in the specs below?

Reply:
There's dozens of [JAC codes](https://www.hesa.ac.uk/support/documentation/jacs/jacs3-detailed) so I've added five examples of each: maths, physics and ineligible. The ones I've used should cover each of the Maths JAC code prefixes (G1, G2, G3, G9) as well as Physics (F3)

Reply:
👍